### ISSUE TITLE
What happened to ssdt unit test documentation

### ISSUE THREAD

**Issue by RubixRevenge** [MicrosoftDocs/visualstudio-docs#1118]
I was about to read the ssdt unit test docs again... And they're gone. Will they be back?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 5416573b-dd99-f3c9-1707-32fb38023788
* Version Independent ID: 45ebd856-0161-4ea2-2833-d1ec165ebc97
* Content: [Visual Studio testing tools - Visual Studio](https://docs.microsoft.com/en-us/visualstudio/test/improve-code-quality)
* Content Source: [docs/test/improve-code-quality.md](https://github.com/MicrosoftDocs/visualstudio-docs/blob/master/docs/test/improve-code-quality.md)
* Product: **visual-studio-dev15**
* GitHub Login: @gewarren
* Microsoft Alias: **gewarren**

**Comment by gewarren** [comment]
@stevestein I can't seem to find SSDT unit test docs either. Can you help? This link (https://msdn.microsoft.com/en-us/library/jj851212(v=vs.103).aspx) redirects to previous-versions but I can't seem to find the docs there.

**Comment by gewarren** [comment]
@RubixRevenge I believe this happened accidentally, and we are hoping to resolve it soon. Thanks for alerting us to this!

**Comment by gewarren** [comment]
@RubixRevenge We have removed the redirects that were causing those topics to "go missing". The links should all be working now. Thank you so much for letting us know!

**Comment by RubixRevenge** [comment]
Google search results are still pointing me to the redirect pages. Assuming
it takes a while for the Google search results to update, how do I search
for the pages I need?

Henry Wyckoff
henry.wyckoff@gmail.com



On Fri, Jun 15, 2018 at 11:40 AM, Genevieve Warren <notifications@github.com

**Comment by gewarren** [comment]
Can you try this link and then navigate from there using the table of contents? https://msdn.microsoft.com/en-us/library/jj851200(v=vs.103).aspx

**Comment by RubixRevenge** [comment]
I"m still getting "Previous versions".

I've already tried the docs.microsoft search without success. And I don't
look forward to crawling through the site -- I've spent enough time on this
as it is. I already have my Google search terms set up for easy access.
I'll check next week in the hope that the Google search results are
refreshed.  If not, I'll drop a line here.


Henry Wyckoff
henry.wyckoff@gmail.com



On Fri, Jun 15, 2018 at 12:23 PM, Genevieve Warren <notifications@github.com
